src/struct/utils/ClientUtil.js

```diff
@@ -177,21 +177,46 @@ class ClientUtil {
             let name = caseSensitive ? e.name : e.name.toLowerCase();
             let t = caseSensitive ? text : text.toLowerCase();
 
-            return name === t || name === t.replace(/:/g, '');
+            return name === t || name === t.replace(/:/g, '') || name.includes(t) || name.includes(t.replace(/:/g, ''));
         };
 
         return emojis.get(text) || emojis.find(check);
     }
 
     /**
+     * Resolves a Guild from a string, such as an ID, or name.
+     * @param {string} text - Text to resolve.
+     * @param {boolean} [caseSensitive=false] - Makes finding by name case sensitive.
+     * @returns {Guild}
+     */
+    resolveGuild(text, caseSensitive = false) {
+        let guilds = client.guilds;
```

Comment:
`this.client.guilds` and remove the regex below because there really isn't a way to mention a guild.
Also I don't like having a space between my parens and braces 😛